supervisor/hyperpod.go

```diff
@@ -123,7 +123,12 @@ func (hp *HyperPod) initPodNetwork(c *Container) error {
 			Ip:     info.Ip,
 		}
 
-		hp.vm.AddNic(info.Index, fmt.Sprintf("eth%d", idx), conf)
+		err = hp.vm.AddNic(info.Index, fmt.Sprintf("eth%d", idx), conf)
+		if err != nil {
+			glog.Error(err)
+			return err
+		}
+
```

Comment:
if the patchset need to be update, this check can be moved back to the patch which add `hp.vm.AddNic(....)`